src/AAVE/Modal/WithdrawModal.jsx

```diff
@@ -436,6 +459,11 @@ return (
                 config,
                 loading: state.loading,
                 children: `Approve ${symbol}`,
+                disabled:
+                  !isValid(state.newHealthFactor) ||
+                  state.newHealthFactor === "" ||
+                  (Big(state.newHealthFactor).lt(1) &&
```

Comment:
Why disable button if `health factor < 1`?

Reply:
When `health factor < 1`, the user has borrowed money. it is easy to be liquidated

Reply:
oh, yeah. that's right.